native/machine-coding/file-uploader/script.js

```diff
@@ -0,0 +1,21 @@
+const form = document.querySelector("form"),
+fileInput = document.querySelector(".file-input"),
+progressArea = document.querySelector(".progress-area"),
+uploadedArea = document.querySelector(".uploaded-area");
```

Comment:
Looks like these two variables (progressArea and uploadedArea) are not used anywhere
You can remove them. Also remove the corresponding html elements and css